### DISCUSSION TITLE
Spliting result image with multiple images

### DISCUSSION THREAD

**Discussion by jopemachine** [lowlighter/metrics#980] [Q&A] [answered]
This awesome library provides a lot of useful plugins, but in my personal view, when I put them all together in one image, some people could feel a little distracted or the image looks too long. 

I think it would be great to split this with multiple images into toggle sections like the below example (Images are omitted other than the `Base information` section and `Notable contribution`).

# About me example
<details>
<summary>Base information</summary>

![jopemachine](https://user-images.githubusercontent.com/18283033/162610942-353ea021-b8a1-4826-9631-7a11e5950937.svg)

</details>

<details>
<summary>Notable Contributions</summary>

![jopemachine (1)](https://user-images.githubusercontent.com/18283033/162611414-0b6f7617-383d-4702-810d-bc997cae1d90.svg)

</details>

<details>
<summary>Achievements</summary>

</details>

<details>
<summary>RSS Feeds</summary>

</details>

<details>
<summary>Fortunes</summary>

</details>

... and others

---

For this form, it would be great for metrics to generate *multiple image options*.
 
I'm wondering your opinion.

**Comment by lowlighter** [comment, marked as answer]
Hi !

Thanks a lot for your feedback 👍 

You should be able to achieve something similar to what you said by using multiple jobs rather than a single one, and using a different filename so they don't conflict together:

```yml
# workflow.yml
- uses: lowlighter/metrics@latest
  with:
    filename: base.svg

- uses: lowlighter/metrics@latest
  with:
    base: ''
    plugin_notable: yes
    filename: notable.svg
```

And use something like this in your markdown:

```html
<!-- README.md -->
<details>
  <summary>Base information</summary>
  <img src="/base.svg">
</details>

<details>
  <summary>Notable contributions</summary>
  <img src="/notable.svg">
</details>
```

Since creating additional jobs may pollute commit history, you can take a look at [Configuring output action](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-configuring-output-action) to avoid it spamming your repositories.

It's also possible to achieve something similar by using the [Markdown template](https://github.com/lowlighter/metrics/blob/master/source/templates/markdown/README.md) instead

Hope it helps 👍
